OriginateTheme/Swift/Models/Components.swift

```diff
@@ -8,9 +8,53 @@
 
 import UIKit
 
-struct Components {
+public struct Components {
     var dictionary: ThemeDefinition
     
+    lazy var navigationBar: NavigationBar = {
+        return NavigationBar(dictionary: self.dictionary)
+    }()
+    
+    public init(dictionary: ThemeDefinition = [:]) {
+        self.dictionary = dictionary
+    }
+}
+
+public struct NavigationBar {
+    var dictionary: ThemeDefinition
+    
+    public var description: UIFont {
```

Comment:
Also here when we merge this PR we shouldn't add those examples here. Let's put them on the README